### PR TITLE
initial

### PR DESCRIPTION
**Pull request by yairvogel**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;static class RandomEquations
```
Comment (shaybenh7) [inline Program.cs]: Different class

Comment (shaybenh7) [inline Program.cs]: new line

File: Program.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        private readonly static char[] op = {'+', '-', '*', '/'};
+
+        private static int Min { get; set; } = 0;
+        private static int Max { get; set; } = 10;
+
+        public static IEnumerable<string> GenerateRandomEquations(int EquationCount = 100)
+        {
+            for(int count = 0; count < EquationCount; count++)
+            {
+                yield return GenerateRandomEquation();
```
Comment (shaybenh7) [inline Program.cs]: nice :)

File: Program.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        {
+            int equationLength = Rand.Next(1, 10);
+            StringBuilder eq = new StringBuilder("", equationLength * 4);
+            for(int i = 0; i < equationLength; i++)
+            {
+                eq.Append(Rand.Next(Min, Max).ToString());
+                char randOp = op[Rand.Next() % 4];
+                eq.Append(new char[] { ' ', randOp, ' '});
+            }
+            eq.Append(Rand.Next(Min, Max).ToString());
```
Comment (shaybenh7) [inline Program.cs]: new line

File: Program.cs [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        }
+    }
+
+namespace OnBoardAssignment
+{
+    class Program
+    {
+        public static void Main(string[] args)
+        {
+            string dirPath = @"C:\Users\t-yvogel\OnboardingAssignment";
```
Comment (shaybenh7) [inline Program.cs:11]: use a global variable

File: Program.cs [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        private static void Generate(int fileAmount, string dirPath)
+        {
+            DirectoryInfo dir = new DirectoryInfo(dirPath);
+            if (!Directory.Exists(dir.FullName))
+            {
+                Directory.CreateDirectory(dir.FullName);
+            }
+
+            // Delete all files in the dir
+            foreach( FileInfo file in dir.EnumerateFiles())
```
Comment (shaybenh7) [inline Program.cs:25]: remove space

Comment (shaybenh7) [inline Program.cs:25]: add space before brackets in all your if\for statements

File: Program.cs (hunk 1 above)
Comment (shaybenh7) [inline Program.cs]: I saw you have a dedicated class for that. why is this here?

File: Program.cs [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+                file.Delete();
+            }
+            // maybe delete dirs as well?
+
+            for(int fileNum = 0; fileNum < fileAmount; fileNum++)
+            {
+                File.WriteAllLines(
+                    dirPath + $@"\{fileNum}.txt",
+                    RandomEquations.GenerateRandomEquations()
+                    );
```
Comment (shaybenh7) [inline Program.cs:36]: RandomEquations.GenerateRandomEquations());
no need for new line

File: Program.cs (hunk 3 above)
Comment (yairvogel) [reply Program.cs]: between the closing moustache and the append statement?

File: Program.cs (hunk 4 above)
Comment (yairvogel) [reply Program.cs:11]: I'll change that into a program argument
